ch8/numb3rs.py

```diff
@@ -0,0 +1,85 @@
+def search(path):
+    """
+    완전탐색
```

Comment:
Python2 does a bad job of handling unicode characters (Korean characters included.) On some systems, this code may fail only because you have Korean characters. I say you put English comments. 

Read this first. https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/ (Btw, Joel Spolsky, the CEO of StackOverflow, writes some informative essays. More essays found on his blog)

Then this. https://stackoverflow.com/questions/4872007/where-does-this-come-from-coding-utf-8

Reply:
changed Korean to English